js/index.js

```diff
@@ -100,10 +100,10 @@ document.addEventListener('DOMContentLoaded', () => {
     });
 
     skillsSection.addEventListener('mouseout', (event) => {
-        if(event.target.tagName == 'LI') {
-            for(let i = 0; i < skills.length; i++) {
-                if(event.target.textContent == skills[i].toUpperCase()) {
-                event.target.textContent = skills[i]
+        if (event.target.tagName == 'LI') {
+            for (let i = 0; i < skills.length; i++) {
+                if (event.target.textContent == skills[i].toUpperCase()) {
```

Comment:
The toUpperCase is a little jarring, and messes with the ui quite a bit, maybe if you add a fixed width on the skills this wont happen. It doesn't matter that much, just something I noticed.

Reply:
thanks for the opinion, didn't think about that this way